mission_control/navigator_alarm/navigator_alarm/alarm_helpers.py

```diff
@@ -25,20 +25,20 @@ def __init__(self):
         # For now, unused - future: Use for system reports
         # self.alarm_spawn_pub = rospy.Publisher('/alarm_spawn', Alarm, latch=True, queue_size=1)
 
-        # Get ROS name
-        self.node_name = rospy.get_name()
```

Comment:
Why not do this?

Should only fail if you try to create the alarm before init'ing the node